### PR TITLE
ci: inject local images to k3d instead of publishing them

### PR DESCRIPTION
**Pull request by ybettan**
This will also remove the need to clean the image registry after each
PR.

Signed-off-by: Yoni Bettan <ybettan@redhat.com>

### CODE REVIEW

File: .github/workflows/e2e-k3s.yaml [hunk 1]
```diff
@@ -9,83 +9,39 @@ on:
...
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
+        run: make build-frontend-amd64
       - name: Create a k3s cluster
         uses: AbsaOSS/k3d-action@v2
         with:
-          cluster-name: "test-cluster"
+          cluster-name: "signoz-cluster"
+      - name: Inject the imaged to the cluster
```
Comment (prashant-shahi) [inline .github/workflows/e2e-k3s.yaml:30]: > `imaged`

can you fix this typo here?

Comment (ybettan) [reply .github/workflows/e2e-k3s.yaml]: Fixed.

File: .github/workflows/e2e-k3s.yaml [hunk 2]
```diff
@@ -9,83 +9,39 @@ on:
 
 jobs:
 
-  image-build-and-push-query-service:
+  e2e-k3s:
     runs-on: ubuntu-latest
     if: ${{ github.event.label.name == 'ok-to-test' }}
```
Comment (prashant-shahi) [inline .github/workflows/e2e-k3s.yaml:14]: @ybettan  can you include `DOCKER_TAG` env for the job instead of adding it in multiple steps?

**Review by prashant-shahi — CHANGES_REQUESTED** [review]
left comments

File: .github/workflows/e2e-k3s.yaml (hunk 2 above)
Comment (ybettan) [reply .github/workflows/e2e-k3s.yaml:14]: Good point, updating it.

**Review by prashant-shahi — APPROVED** [review]
Looks good to me :shipit:

**Pull request merged**